mindsdb/integrations/libs/ml_handler_process/learn_process.py

```diff
@@ -61,11 +61,9 @@ def learn_process(data_integration_ref: dict, problem_definition: dict, fetch_da
                         )
                     )
                     sqlquery = SQLQuery(query, session=sql_session)
-                elif data_integration_ref['type'] == 'view':
```

Comment:
Maybe keep this for back compatibility?

Reply:
It won't be used because it removed [here](https://github.com/mindsdb/mindsdb/pull/9135/files#diff-ce5f87764f401f9a2a6eeb930f777d863ea3d6985a3b948d0361dddbbba192abL229)

Reply:
It will break existing 'predictors' with `data_integration_ref->>'type' = 'view'`

Reply:
ok, returned 'type=view' block